numpy/lib/function_base.py

```diff
@@ -4008,7 +4008,7 @@ def trapz(y, x=None, dx=1.0, axis=-1):
     """
     Integrate along the given axis using the composite trapezoidal rule.
 
-    Integrate `y` (`x`) along given axis.
+    Integrate `y` (`x`) along given axis, in the order of values passed.
```

Comment:
Wording this is tricky, maybe worth mentioning this only applies if `x` is given.
```suggestion
    Integrate `y` (`x`) along given axis. If `x` is provided, the integration happens in sequence along its elements - they are not sorted.
```

Could also embed some latex here (if you replace `"""` with `r"""` above) with

```suggestion
    Along each 1d slice on the given axis, compute :math:`\int y(x) dx`.
    When `x` is specified, this integrates along the parametric curve, computing :math:`\int_t y(t) dt = \int_t y(t) \left.\frac{dx}{dt}\right|_{x=x(t)} dt`.
```

Let's see what other reviewers have to say.